dashboard/context_processors.py

```diff
@@ -5,16 +5,6 @@
 logger = logging.getLogger(__name__)
 
 
-def current_user_course_id(request):
-    logger.info(current_user_course_id.__name__)
-    course_id = str(request.resolver_match.kwargs.get('course_id'))
-    if not course_id:
-        logger.info("Course ID could not be determined from request, attempting to look up for user {}".format(
-            request.user.username))
-        course_id = db_util.get_default_user_course_id(request.user.username)
-    return {'current_user_course_id': course_id}
```

Comment:
This is redundant as with React flow there is no reload of page React routers control the navigation no BE call to this code is made unless user accesses the /grades view etc from the browser window.